src/navigations/AppNavigator/index.tsx

```diff
@@ -13,7 +13,7 @@ export const RootStack = createNativeStackNavigator<RootStackParamList>()
 
 const AppNavigator: React.FC = () => {
   const screenForAppState = useCallback(() => {
-    return [SignIn(), MainBottonNavigator()]
+    return [MainBottonNavigator(), Activities()]
```

Comment:
It is missing the Sign In group! Please fix it

Reply:
shut up and merge it

Reply:
No fix, no merge. Please, fix it.

Reply:
no

Reply:
merge it